pkg/config/controller_config.go

```diff
@@ -68,6 +71,13 @@ type ControllerConfig struct {
 	TargetGroupBindingMaxConcurrentReconciles int
 	// Max exponential backoff delay for reconcile failures of TargetGroupBinding
 	TargetGroupBindingMaxExponentialBackoffDelay time.Duration
+
+	// EnableBackendSecurityGroup specifies whether to use optimized security group rules
+	EnableBackendSecurityGroup bool
+
+	// BackendSecurityGroups specifies the configured backend security groups to use
+	// for optimized security group rules
+	BackendSecurityGroups []string
```

Comment:
what's the use case of multiple backSG vs a single one?

Reply:
updated in the new commit.